### ISSUE TITLE
I need help

### ISSUE THREAD

**Issue by Goaty75** [finiteCraft/finiteCraft#2]
I am as unexperienced as physically possible with python, so I need help running this. Thanks to anyone who helps!

**Comment by quantumbagel** [comment]
Hi!

This project will eventually have a GitHub Pages frontend at [finitecraft.github.io](finitecraft.github.io). This code will run on my private Raspberry Pi (crafter.py) and post it to the api repository on the account. If you still want to run finitecraft (I'd love to have a more powerful server!), we are working on a Docker image that will set itself up and run in an independent container.

I'll leave this issue open until we have a guide for setup ready. 

Thanks for your interest!

**Comment by Goaty75** [comment]
Also, when you discover an element and it's an fd, when you make it, does it register as an fd? Just want to know ahead of time.

**Comment by quantumbagel** [comment]
Yes, it's in the JSON from the generated code. See as an example: https://github.com/finiteCraft/api/blob/master/display/0.json

We don't have any yet :(

**Comment by Goaty75** [comment]
Maybe you should try challenges for the bot to see if it's really that good like "Orange Grayhound Worm Meal" or something

**Comment by quantumbagel** [comment]
I've added an install guide to the README. Let me know if that helps! (I've also made it much easier to run :P)

**Comment by Goaty75** [comment]
So I installed it, how do I use it?

**Comment by quantumbagel** [comment]
Right now, finiteCraft has three components:

- The scraper. This saves the craft data to the MongoDB instance, which you can review by connecting to the server at 172.21.0.2. This is complete
- The API. This will allow our web server to acquire information from the database securely. We are working on this now.
- The web server. This will display the information in a human readable format using the API

TLDR: it's not human readable yet
